crates/bevy_ecs/src/change_detection.rs

```diff
@@ -735,6 +735,34 @@ impl<'a> MutUntyped<'a> {
         self.value.as_ref()
     }
 
+    /// Turn this [`MutUntyped`] into a [`Mut`] by mapping the inner [`PtrMut`] to another value,
+    /// without flagging a change. [`MutUntyped`] equivalent of [`Mut::map_unchanged`].
```

Comment:
```suggestion
    /// without flagging a change. 
    /// This function is the untyped equivalent of [`Mut::map_unchanged`].
```

This phrasing reads better IMO.